src/PowerShellEditorServices/Services/Symbols/ScriptDocumentSymbolProvider.cs

```diff
@@ -15,9 +16,11 @@ namespace Microsoft.PowerShell.EditorServices.Services.Symbols
     /// Provides an IDocumentSymbolProvider implementation for
     /// enumerating symbols in script (.psd1, .psm1) files.
     /// </summary>
-    public class ScriptDocumentSymbolProvider : IDocumentSymbolProvider
+    internal class ScriptDocumentSymbolProvider : IDocumentSymbolProvider
     {
-        IEnumerable<SymbolReference> IDocumentSymbolProvider.ProvideDocumentSymbols(
+        string IDocumentSymbolProvider.ProviderId => nameof(ScriptDocumentSymbolProvider);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private property 'IDocumentSymbolProvider.ProviderId'.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4979332)